### PR TITLE
Refactor/workspace (tests): update tests for refactored Workspace unlinked pages

### PR DESCRIPTION
**Pull request by gweiying**
This PR updates tests for the refactored Workspace flow introduced in #648.

## Details
In this PR:
- Updates existing Workspace tests for the new endpoints and url params
- Adds new tests for:
  - Moving pages from Workspace to subfolder/folder
  - Creating and editing pages in Workspace with foreign characters

## Deploy notes
Please update your env variable `E2E_COMMIT_HASH` to the following:
```export E2E_COMMIT_HASH=c80b66da287adde4424a2b4a28ad3c0db9018ef9```

### CODE REVIEW

File: cypress/integration/editPage.spec.js [hunk 1]
```diff
@@ -196,7 +199,7 @@ describe("Edit unlinked page", () => {
     cy.get("#modal-delete").click()
 
     // Assert: page no longer exists
-    cy.visit(`/sites/${TEST_REPO_NAME}/pages/${TEST_UNLINKED_PAGE_FILE_NAME}`)
+    cy.visit(`/sites/${TEST_REPO_NAME}/editPage/${TEST_PAGE_TITLE_ENCODED}`)
     cy.contains("The page you are looking for does not exist anymore.")
```
Comment (alexanderleegs) [inline cypress/integration/editPage.spec.js:203]: Could we look into increasing the timeout for this test? It's consistently failing for me

File: cypress/integration/move.spec.js [hunk 2]
```diff
@@ -16,9 +16,144 @@ describe("Move flow", () => {
...
+    beforeEach(() => {
+      cy.setCookie(COOKIE_NAME, COOKIE_VALUE)
+      window.localStorage.setItem("userId", "test")
+      cy.visit(`${CMS_BASEURL}/sites/${TEST_REPO_NAME}/workspace`)
+    })
+
+    it("Should be able to navigate from Workspace to subfolder back to Workspace via MoveModal buttons", () => {
+      cy.contains(FILENAME_WORKSPACE_TO_FOLDER).should("exist")
+      cy.get(
+        `button[id^="folderItem-dropdown-${FILENAME_WORKSPACE_TO_FOLDER}"]`
```
Comment (alexanderleegs) [inline cypress/integration/move.spec.js:39]: Should this be `button[id^="pageCard-dropdown-${FILENAME_WORKSPACE_TO_FOLDER}"]` instead? Few other instances of this in the move test

**Review by alexanderleegs — CHANGES_REQUESTED** [review]
Left some comments

File: cypress/integration/move.spec.js (hunk 2 above)
Comment (gweiying) [reply cypress/integration/move.spec.js:39]: good catch - fixed in c50f7bd

File: cypress/integration/editPage.spec.js (hunk 1 above)
Comment (gweiying) [reply cypress/integration/editPage.spec.js:203]: good idea! fixed here 6b62b78

**Review by alexanderleegs — APPROVED** [review]
lgtm

**Pull request merged**